### PR TITLE
Feat/cdn/cache

### PR DESCRIPTION
**Pull request by sguiheux**
1. Description
1. Related issues
1. About tests
1. Mentions

### CODE REVIEW

File: engine/cdn/item_logs_handler.go [hunk 1]
```diff
@@ -55,61 +54,11 @@ func (s *Service) getItemLogsDownloadHandler() service.Handler {
 			return sdk.WithStack(sdk.ErrNotFound)
 		}
 
-		// Try to load item and item units for given api ref
-		item, err := index.LoadItemByAPIRefHashAndType(ctx, s.Mapper, s.mustDBWithCtx(ctx), token.APIRefHash, itemType)
+		rc, err := s.getItemLogValue(ctx, itemType, token.APIRefHash, 0, 100000)
```
Comment (richardlt) [inline engine/cdn/item_logs_handler.go:57]: 100000?

File: engine/cdn/types.go [hunk 2]
```diff
@@ -44,8 +46,9 @@ type Configuration struct {
 	EnableLogProcessing bool                                   `toml:"enableLogProcessing" comment:"Enable CDN preview feature that will index logs (this require a database)" json:"enableDatabaseFeatures"`
 	Database            database.DBConfigurationWithEncryption `toml:"database" comment:"################################\n Postgresql Database settings \n###############################" json:"database"`
 	Cache               struct {
-		TTL   int `toml:"ttl" default:"60" json:"ttl"`
-		Redis struct {
+		TTL     int   `toml:"ttl" default:"60" json:"ttl"`
+		LruSize int64 `toml:"lruSize" default:"512000000" json:"lruSize"`
```
Comment (richardlt) [inline engine/cdn/types.go:50]: Can we have an option to disable it by default ? Or lower size?

File: engine/cdn/lru/cdn_lru.go [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+	"github.com/ovh/cds/engine/cache"
+	"github.com/ovh/cds/sdk/log"
+)
+
+type AbstractLRU struct {
+	maxSize int64
+	db      *gorp.DbMap
+}
+
+type Interface interface {
```
Comment (richardlt) [inline engine/cdn/lru/cdn_lru.go:19]: Do we really need this interface ?

**Review by richardlt — APPROVED** [review]

File: engine/cdn/lru/cdn_lru.go (hunk 3 above)
Comment (sguiheux) [reply engine/cdn/lru/cdn_lru.go:19]: Interface = life

**Pull request merged**